Gemfile

```diff
@@ -21,6 +21,8 @@ else
   gem 'rake', '>= 12.3.3'
 end
 
+gem 'diff-lcs', ENV.fetch('DIFF_LCS_VERSION', '~> 1.4')
```

Comment:
Don't we want to stick to old flag like `if RUBY_VERSION`?

Reply:
If diff-lcs 1.x fails on < Ruby 2.5 it’s _still_ a failure, even though I’m no longer running CI on those versions.

Reply:
Oups sorry. I just saw "I'm going to setup CI to test both versions of diff-lcs."